web/style/transvision.css

```diff
@@ -57,20 +41,24 @@ html {
 
 body {
     color: #333;
-    font-family: 'Open Sans', Arial, sans-serif;
-    font-size: 14px;
+    font-family: 'Fira Sans Light', Arial, sans-serif;
+    font-size: 0.9em;
     line-height: 1.5;
     margin: 0 auto;
     padding: 0;
-    border-top: 1px solid #fff;
     position: relative;
-    background-color: #cadded;
-    background-image: url(/img/grain.png), linear-gradient(to bottom, #cadded 0%, #f6f4ec 100%);
-    background-origin: padding-box, padding-box;
-    background-position: 0 0, 0 0, 0 0;
-    background-repeat: repeat;
-    background-size: auto auto, auto auto, auto auto;
     min-width: 320px;
+    display: flex;
+    min-height: 100vh;
```

Comment:
Do we really want to use something that is not even supported in the current Firefox ESR?
http://caniuse.com/#feat=viewport-units